flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -276,6 +276,13 @@ class SentryFlutterOptions extends SentryOptions {
   /// Defaults to `true`
   bool enableFramesTracking = true;
 
+  /// The period (in minutes) for which to retain frame information.
+  /// Used to calculate the maximum number of frames to store for active spans.
+  /// If the number of frames exceeds the maximum, the frame tracking is cancelled
+  /// and frame metrics are cleared for the current active spans.
+  /// Default value is 3 minutes.
+  int framesTrackingRetentionPeriod = 3;
```

Comment:
Why don't you use `Duration`? That way you will not have any confusion regarding the unit:

```suggestion
  Duration framesTrackingRetentionPeriod = Duration(minutes: 3);
```

Reply:
I think I'll just use a fixed constant, it's a pretty confusing option tbh